projects/igniteui-angular/src/lib/icon/icon.service.ts

```diff
@@ -90,6 +91,7 @@ export class IgxIconService {
         }
     }
 
+
```

Comment:
Would appreciate it greatly if those are cleaned out so no changes are made to this file without need :)